iexec-sms-library/src/main/java/com/iexec/sms/SmsClient.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
`com.iexec.sms.SmsClient`
:arrow_down: 
`com.iexec.sms.api.SmsClient`
?

Reply:
`com.iexec.sms.client.SmsClient`?

Reply:
I see your point, but is there really a meaning in creating a dedicated package for a single class ?
I really don't have a decisive answer on this and am ready to change it if the team prefer a package.

Reply:
Wanted to prepare architecture for next increment to avoid renamings in clients, but you can keep as is if you prefer.

(future note: client is good to me)

Reply:
Taking into consideration next increment with a service class, do you want to have a `client` or `api` package containing a single class ?

Reply:
client is good